ssr-playground/package.json

```diff
@@ -0,0 +1,34 @@
+{
+    "private": true,
+    "name": "@lwc/ssr-playground",
+    "version": "3.0.0",
+    "type": "module",
+    "description": "Playground project to experiment with LWC.",
+    "scripts": {
+        "dev": "node ./server.js",
```

Comment:
Without a `"build" script, NX doesn't actually build this project. Should it have a prod-mode `build` like the other `playground`?